spacemacs/extensions/spacemacs-theme/README.md

```diff
@@ -1,6 +1,6 @@
 # Spacemacs-theme
 
-![spacemacs-theme](/../screenshots/spacemacs-theme-screenshot.png?raw=true)
+![spacemacs-theme](https://github.com/nashamri/spacemacs-theme/blob/screenshots/spacemacs-theme-screenshot.png?raw=true)
```

Comment:
Yeah maybe using it as a sub-module is the cleaner approach. Anyways, I think `gnus` is the last mode that I need to support before I try to submit it to MELPA